### ISSUE TITLE
 OSError: [WinError 193] %1 is not a valid Win32 application Ask Question

### ISSUE THREAD

**Issue by Poolitzer** [sdispater/poetry#1317]
- [x] I am on the [latest](https://github.com/sdispater/poetry/releases/latest) Poetry version.
- [x] I have searched the [issues](https://github.com/sdispater/poetry/issues) of this repo and believe that this is not a duplicate.
- [x] If an exception occurs when executing a command, I executed it again in debug mode (`-vvv` option).

- **OS version and name**: Windows 10
- **Poetry version**: 0.12.17
- **Link of the [file](https://gist.github.com/) with the contents of my pyproject.toml file**: https://github.com/Poolitzer/sticker-finder/blob/master/pyproject.toml

## Issue
Hey, I think there is a problem with using windows and the run command, because you don't specify the executable. This happens, because run calls execute [see here](https://github.com/sdispater/poetry/blob/f4803e91bca440e6f4af6ad54e396d01c024a76f/poetry/console/commands/run.py#L20) [and then here](https://github.com/sdispater/poetry/blob/f4803e91bca440e6f4af6ad54e396d01c024a76f/poetry/utils/env.py#L392) without any extra windows care.

**Comment by samuelpetroline** [comment]
Is there any way to fix this? I haven't been able to find a workaround

**Comment by finswimmer** [comment]
Hello @samuelpetroline ,

could you please add some more information to the issue. How can one reproduce it?

Thanks.

fin swimmer

**Comment by mardiros** [comment]
Hi,

I encoutered the same bug

what I did:

```
pip install poetry

C:\blah\ poetry new myprog
works...
C:\blah\ poetry add
works...
C:\blah\ poetry run .\myprod\whatever.py

[OSError]
[WinError 193] %1 is not a valid Win32 application

```

So I uninstall it, try with poetry 0.12

Then I read the doc and do preferred / documented way

(Invoke-WebRequest -Uri https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py -UseBasicParsing).Content | python


For information, I am a quite long time poetry user on Linux, and try it on windows with on my son's 
computer laptop.


I've done all the things in `vscode` Terminal. I don't know if that matter, I am not an advanced windows user.
